### PR TITLE
intro 2 TH RC2 with react to Taylor

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

File: aspnet/mvc/views/tag-helpers/intro.rst [hunk 1]
```diff
@@ -96,19 +103,26 @@ Opting out of individual elements
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ``@tagHelperPrefix`` directive allows you to specify a tag prefix string to enable Tag Helper support and to make Tag Helper usage explicit. In the code image below, the Tag Helper prefix is set to "th:", so only those elements using the prefix "th:" support Tag Helpers (Tag Helper-enabled elements have a distinctive font). The ``<label>`` and ``<span>`` elements have the Tag Helper prefix and are Tag Helper-enabled, while the ``<input>`` element does not.
+The ``@tagHelperPrefix`` directive allows you to specify a tag prefix string to enable Tag Helper support and to make Tag Helper usage explicit. In the code image below, the Tag Helper prefix is set to ``th:``, so only those elements using the prefix ``th:`` support Tag Helpers (Tag Helper-enabled elements have a distinctive font). The ``<label>`` and ``<input>`` elements have the Tag Helper prefix and are Tag Helper-enabled, while the ``<span>`` element does not.
 
 .. image:: intro/_static/thp.png 
 
+:Note: Quotes are optional with the ``@tagHelperPrefix`` directive. The following two directives are equivalent:
```
Comment (Rick-Anderson) [inline aspnet/mvc/views/tag-helpers/intro.rst]: @NTaylorMullen "th:" (quotes) doesn't work in build 117
Will it work for public RC2 - tooling preview or should I remove?

Comment (Rick-Anderson) [inline aspnet/mvc/views/tag-helpers/intro.rst]: @NTaylorMullen  sorry had to create a new PR - I fixed the image as you suggested and I removed all the tabs (so there is a lot of noise)

Comment (NTaylorMullen) [inline aspnet/mvc/views/tag-helpers/intro.rst]: Wont work consistently for rc2, remove.

**Comment by Rick-Anderson** [comment]
@NTaylorMullen can I get a ship-it?

**Comment by NTaylorMullen** [comment]
@Rick-Anderson I see a ton of deletions in this PR. What are you removing?

**Comment by Rick-Anderson** [comment]
I only refer to one file in this directory ( and a few files in the Authoring Tag Helpers directory) So I removed the two projects 

- The ASP.NET 5 Web app with individual user accounts
- The ASP.NET 4.x Web app with individual user accounts (using HTML Helpers) - But I never refer to the code, just screen shots

**Comment by NTaylorMullen** [comment]
:shipit:

**Pull request merged**